modules/widgets/src/fullscreen-widget.ts

```diff
@@ -0,0 +1,55 @@
+import type {Deck, Widget} from '@deck.gl/core';
+
+/* global document */
+const defaultStyle: Partial<CSSStyleDeclaration> = {
```

Comment:
Any value in making this a static on the class? 
```suggestion
const defaultStyle: Readonly<Partial<CSSStyleDeclaration>> = {
```